src/components/swapv2/LimitOrder/ListOrder/index.tsx

```diff
@@ -0,0 +1,555 @@
+        ...response,
+        type: TRANSACTION_TYPE.CANCEL_LIMIT_ORDER,
+        summary: order
+          ? `Order ${formatAmountOrder(order.makingAmount)} ${order.makerAssetSymbol} to ${formatAmountOrder(
+              order.takingAmount,
+            )} ${order.takerAssetSymbol}`
+          : `all orders`,
+      })
+    return
+  }
+
+  const onCancelOrder = async (order: LimitOrder | undefined) => {
+    try {
+      await requestCancelOrder(order)
+      setFlowState(state => ({ ...state, showConfirm: false }))
+    } catch (error) {
+      console.error(error)
+      setFlowState(state => ({
+        ...state,
+        attemptingTxn: false,
+        errorMessage: 'Error occur. Please try again.',
+      }))
+    }
+  }
+
+  const onCancelAllOrder = () => {
+    showConfirmCancel()
+    setIsCancelAll(true)
+  }
+
+  const onUpdateOrder = async () => {
+    await requestCancelOrder(currentOrder)
+  }
+
+  const disabledBtnCancelAll = totalOrderNotCancelling === 0
+  const isTabActive = isActiveStatus(orderType)
+
+  useEffect(() => {
+    const orderCancelling = orders.length - totalOrderNotCancelling
+    window.onbeforeunload = () => (orderCancelling > 0 && ordersUpdating.length > 0 ? '' : null)
+  }, [totalOrderNotCancelling, orders, ordersUpdating])
+
+  return (
+    <>
+      <Flex justifyContent={'space-between'} alignItems="center">
+        <TabSelector
+          setActiveTab={onSelectTab}
+          activeTab={isTabActive ? LimitOrderStatus.ACTIVE : LimitOrderStatus.CLOSED}
+        />
+      </Flex>
+
+      <Flex flexDirection={'column'} style={{ gap: '1rem' }}>
+        <SearchFilter>
+          <SelectFilter
+            key={orderType}
+            options={isTabActive ? ACTIVE_ORDER_OPTIONS : CLOSE_ORDER_OPTIONS}
+            value={orderType}
+            onChange={setOrderType}
+          />
+          <SearchInputWrapped
+            placeholder={t`Search by token symbol or token address`}
+            maxLength={255}
+            value={keyword}
+            onChange={setKeyword}
+          />
+        </SearchFilter>
+        {loading ? (
+          <LocalLoader />
+        ) : (
+          <>
+            <div>
+              <TableHeader />
+              <ListWrapper>
+                {orders.map((order, index) => (
+                  <OrderItem
+                    isOrderCancelling={isOrderCancelling}
+                    index={index + (curPage - 1) * PAGE_SIZE}
+                    key={order.id}
+                    order={order}
+                    onCancelOrder={showConfirmCancel}
+                    onEditOrder={showEditOrderModal}
+                  />
+                ))}
+              </ListWrapper>
+            </div>
+            {orders.length === 0 && (
+              <NoResultWrapper>
+                <Info size={isMobile ? 40 : 48} />
+                <Text marginTop={'10px'}>
+                  <Trans>You don&apos;t have any {isTabActive ? t`active` : t`history`} orders yet</Trans>
```

Comment:
It will be better to avoid using if in the middle of `<Trans>` as much as possible since it causing hard to translate. It should be 
```
{isTabActive ? <Trans>... active ...</Trans> : <Trans>... history ...</Trans>
```